src/app/clusters/scenes/scenes.c

```diff
@@ -998,8 +998,8 @@ bool emberAfPluginScenesServerParseAddScene(const EmberAfClusterCommand * cmd, u
 
 bool emberAfPluginScenesServerParseViewScene(const EmberAfClusterCommand * cmd, uint16_t groupId, uint8_t sceneId)
 {
-    EmberAfSceneTableEntry entry;
-    EmberAfStatus status = EMBER_ZCL_STATUS_NOT_FOUND;
+    EmberAfSceneTableEntry entry = {};
```

Comment:
Was the lack of an initializer there triggering compiler or static analysis warnings?

Reply:
Compiler warnings.